backend/src/errors.rs

```diff
@@ -0,0 +1,13 @@
+error_chain! {
+    links {
+    }
+
+    foreign_links {
+        ::std::io::Error, Io, "io error";
+        ::json::Error, Json, "json error";
+        ::chrono::ParseError, Chrono, "chrono parsing error";
+    }
+
+    errors {
```

Comment:
One of the next steps would be to go through all the potential error cases and give them a good error message, I guess. This would resolve #6, too, if I follow what that's asking for. Though resolving that may need work on the client side as well.